tpl/js-templates.php

```diff
@@ -5,66 +5,70 @@
 
 <script type="text/template" id="siteorigin-panels-builder">
 
-	<div class="so-builder-toolbar">
+	<div class="siteorigin-panels-builder">
 
-		<a href="#" class="so-tool-button so-widget-add">
-			<span class="so-panels-icon so-panels-icon-plus"></span>
-			<span class="so-button-text"><?php esc_attr_e('Add Widget', 'siteorigin-panels') ?></span>
-		</a>
+		<div class="so-builder-toolbar">
 
-		<a href="#" class="so-tool-button so-row-add">
-			<span class="so-panels-icon so-panels-icon-columns"></span>
-			<span class="so-button-text"><?php esc_attr_e('Add Row', 'siteorigin-panels') ?></span>
-		</a>
-
-		<a href="#" class="so-tool-button so-prebuilt-add">
-			<span class="so-panels-icon so-panels-icon-cubes"></span>
-			<span class="so-button-text"><?php esc_attr_e('Prebuilt', 'siteorigin-panels') ?></span>
-		</a>
-
-		<?php if( !empty($post) ) : ?>
+			<a href="#" class="so-tool-button so-widget-add">
+				<span class="so-panels-icon so-panels-icon-plus"></span>
+				<span class="so-button-text"><?php esc_attr_e('Add Widget', 'siteorigin-panels') ?></span>
```

Comment:
Aaah yeah, strictly this should be `esc_html_e`. That's a slightly better escaping to use where there shouldn't be any HTML added in the translation. `_e` would definitely work though, and probably more than secure enough.